response_operations_ui/controllers/sample_controllers.py

```diff
@@ -59,10 +57,13 @@ def search_samples_by_postcode(postcode) -> dict:
     response = requests.get(url=url,
                             auth=app.config['SAMPLE_AUTH'],
                             params={'postcode': postcode})
+
+    logger.debug("Searching for samples by postcode %s" % postcode)
```

Comment:
Should be: `logger.debug("Searching for samples by postcode", postcode=postcode)`